CMakeLists.txt

```diff
@@ -38,6 +37,18 @@ set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake_modules)
 include(LookUp-GreatCMakeCookOff)
 include(PackageLookup)
 
+if(pythonBindings)
+  include(${CMAKE_SCRIPTS}/AllPythonBindings.cmake)
+endif(pythonBindings)
+
```

Comment:
I guess the stuff below should be inside the if(pythombindings) too. Otherwise PYTHON_INTERP_PREFIX will likely be empty